setup.py

```diff
@@ -96,7 +96,7 @@ def write_git_revision(package_name):
           "codepy>=2017.1",
           "colorama",
           "Mako",
-          "pyrsistent",
+          "pyrsistent<0.19",
```

Comment:
FYI: There are two things that I think would improve this change.

- Add a link explaining why it's there.
- Write the constraint as `!=0.19.1` or some such, where the version is the one including the regression. That way, you don't preclude future updates when there's no reason to do so. (But, to be fair, you might have to update the constraint if further versions show up that include the regression.)